proxy/src/serve_tasks.rs

```diff
@@ -84,7 +84,10 @@ async fn handler_tasks(
 async fn sign_request(req: Request<Body>, config: &config_proxy::Config, target_uri: &Uri, sender: &AppId) -> Result<Request<Body>, (StatusCode, &'static str)> {
     let (mut parts, body) = req.into_parts();
     let body = body::to_bytes(body).await
-        .map_err(|_| ERR_BODY)?;
+        .map_err(|e|{
+            warn!("Unable to read message body: {e}");
+            ERR_BODY
+        })?;
```

Comment:
Thank you for the more detailed error feedback. This was a bit of a pain before.